ISAAR.MSolve.SamplesConsole/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Thrower package is not in the library, please remove or explain why it is needed so that it can be incorporated.